src/components/table/table.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, h, Prop, Element } from '@stencil/core';
+import TabulatorTables from 'tabulator-tables';
+
+export interface Column {
+    title: string;
+    field: string;
+}
+@Component({
+    tag: 'limel-table',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class Table {
+    /**
+     * table data to be displayed
+     */
+    @Prop()
+    public tableData: any[] = [];
```

Comment:
What will happen if the consumer updates the data that is being passes in here, will the table display the new data?